src/frontend/src/components/Instruction.jsx

```diff
@@ -117,6 +117,17 @@ function Instruction() {
                   using the <b>Save Annotation</b> button.
                 </p>
               </li>
+              <li>
+                <strong className="download-annotation">
+                  Download Annotation
+                </strong>
+                : Clicking this button download the current annotation layers
```

Comment:
Just to be a little more clear about what is actually downloaded.
```suggestion
                : Click this button to download uniformly sampled points from the current annotation layers
```